composer.json

```diff
@@ -15,11 +15,16 @@
 		"stecman/symfony-console-completion": "^0.6.1"
 	},
 	"require-dev": {
-		"aik099/coding-standard": "dev-master"
+		"aik099/coding-standard": "dev-master",
+		"phpunit/phpunit": "^4.8 || ^5.7 || ^6.5 || ^7.0"
```

Comment:
Please write an equivalent in `.travis.yml` (see above comment for an idea).

I prefer single PHPUnit version installed globally (the Travis CI does that as well), rather than making different PHPUnit version as dev dependency for every project with tests (waste of disk space IMO).

Reply:
Perhaps we can set the `phpuint` for `^4.8` version.

Reply:
I think we can remove the `phpunit/phpunit` package and using the default PHPUnit version in Travis CI build.

Reply:
> I think we can remove the `phpunit/phpunit` package and using the default PHPUnit version in Travis CI build.

That is what I've said in my other comment (see above). If that works out (wasn't working out of the box for me last time I've tried), then it would be great.